docs/_src/usage/usage/domain_adaptation.md

```diff
@@ -69,7 +69,73 @@ Annotate also supports two different workflows:
 
 ## User Feedback
 
-A simpler and faster process to retrain models is to utilise user feedback.
-Users can give thumbs up or thumbs down to search results through the Rest API
-and these labels will be stored in the `DocumentStore`
-where they can already be used to retrain the `Reader`.
+A simpler and faster process to retrain models is to utilise user feedback. 
+Dedicated annotation work can be costly and time consuming 
+but user feedback is an efficient alternative since it allows for labels to be generated by users, for users, 
+all while the system is already in production.
+We, for example, have used a simple thumbs up / down system in our demos to allow
+users to give feedback.
+
+![image](../../img/demo.png)
+
+Through the Rest API, users can annotate each Haystack result as being either:
+
+* Correct
+* Correct document but wrong text
+* Wrong document and wrong text
+
+To get started, follow these steps:
+
+* Start up the REST API
+    * The simplest way to do this is to call `docker-compose up` from the root directory of the Haystack repository
+    * Alternatively, run `gunicorn rest_api.application:app -b 0.0.0.0:8000 -k uvicorn.workers.UvicornWorker -t 300`
+* Make a POST request to the `doc-qa-feedback` endpoint in the following format:
+```
+{
+  "question": "string",
+  "is_correct_answer": true,
+  "document_id": "string",
+  "model_id": 0,
+  "is_correct_document": true,
+  "answer": "string",
+  "offset_start_in_doc": 0
+}
+```
```

Comment:
Add a line: 

"You find all values you need for the above JSON in the response from your previously executed search request (using the `doc-qa` endpoint)"